collections/hub/golden_path/feature_tests/block_transfer.json

```diff
@@ -7,6 +7,629 @@
       "meta": {
         "info": "Block Transfer"
       },
+      "requests": [
+        {
+          "id": 1,
+          "description": "Get Settlement account balance",
+          "meta": {
+            "info": "Get Settlement account balance"
+          },
+          "apiVersion": {
+            "minorVersion": 0,
+            "majorVersion": 1,
+            "type": "central_admin"
+          },
+          "operationPath": "/participants/{name}/accounts",
+          "path": "/participants/{$inputs.fromFspId}/accounts",
+          "method": "get",
+          "params": {
+            "name": "{$inputs.fromFspId}"
+          },
+          "headers": {
+            "Content-Type": "application/json",
+            "FSPIOP-Source": "{$inputs.hub_operator}"
+          },
+          "url": "{$inputs.HOST_CENTRAL_LEDGER}",
+          "tests": {
+            "assertions": [
+              {
+                "id": 1,
+                "description": "status to be 200",
+                "exec": [
+                  "expect(response.status).to.equal(200)"
+                ]
+              }
+            ]
+          },
+          "delay": "500",
+          "scripts": {
+            "postRequest": {
+              "exec": [
+                "const jsonData = pm.response.body",
```

Comment:
@vijayg10 ... I see that in some places pm.response.body is used and in other places response.body is used. Can we please be consistent if that is not a blocker?

Reply:
Hi @sri-miriyala , pm.response is for postman scripting and response.body should be used for java script.